last-donation/incentive-config.html

```diff
@@ -0,0 +1,17 @@
+<html>
+	<head>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+		<script src="../elt.js"></script>
+		<script src="last-donation-settings.js"></script>
+	</head>
+
+	<body>
+		<div id="incentivePage" class="elt">
+			<div id="form"></div>
```

Comment:
So here's my longer-term plan:
`Incentives` will be a `UL` with checkboxes showing just the Name and (if present) incentive icon.  Checking the box will add incentive's Object to the config displayed in `configOutput`, unchecking will remove it.

But it's late and I'm bad at JS, so this is a short-term workaround.  It reads the user(s) from the widget config, so the `form` div can probably go away.